src/components/keypad/index.js

```diff
@@ -41,6 +59,9 @@ export default class PreAlgebraKeypad extends React.Component<Props, State> {
                 {selectedPage === "Operators" && (
                     <PreAlgebraPage onClickKey={onClickKey} />
                 )}
+                {selectedPage === "Geometry" && (
```

Comment:
This geometry/trigonometry naming situation looks really confusing to me. Is there any way we could simplify it so the page and the prop and the component are the same?

Reply:
I think this will get simplified over time as we work through the variations in the design for this keypad. I think it'll be simpler in the end but I'd be happy to pair on figuring it out together.